index.bs

```diff
@@ -3406,6 +3408,58 @@ The [=remote end steps=] with |command parameters| are:
 
 </div>
 
+#### The browsingContext.setUserAgent Command ####  {#command-browsingContext-setUserAgent}
+
+The <dfn export for=commands>browsingContext.setUserAgent</dfn> command sets the
+{{NavigatorID/userAgent}} string for a browsing context.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      browsingContext.SetUserAgent = (
+        method: "browsingContext.setUserAgent",
+        params: browsingContext.SetUserAgentParameters
+      )
+
+      browsingContext.SetUserAgentParameters = {
+        context: browsingContext.BrowsingContext,
+        ? userAgent: text,
```

Comment:
```suggestion
        userAgent: text / null,
```

Reply:
I was a bit unsure of this. @jgraham could you double check here? Is the preferred pattern to use `?` or `/ null`?

Reply:
The prose checks for null so it needs to be null to match the current text. In general, we tend to use null if we unset the value to the default and undefined (?) if we want it to remain unchanged (e.g., we applied this to the setViewport command).

Reply:
Done.